aries-site/src/examples/tokens/StrongBackgrounds.js

```diff
@@ -0,0 +1,48 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import { Box, Button, Text } from 'grommet';
+import { Add } from 'grommet-icons';
+
+export const StrongBackgrounds = () => {
+  const theme = useContext(ThemeContext);
+  console.log(theme);
+  return (
+    <Box gap="large">
+      <Box
+        background={{
+          color: 'background-neutral-xstrong',
+          dark: theme.dark,
+        }}
+        pad="medium"
+        round="medium"
+        gap="medium"
+      >
+        <Text color="text-onStrong" size="large" weight={500}>
+          color.background.neutral.xstrong
+        </Text>
+        <Box gap="small">
+          <Box direction="row" gap="small">
+            <Text color="text-onStrong" weight={500}>
+              Aa
+            </Text>
+            <Text color="text-onStrong">color.text.onStrong</Text>
+          </Box>
+          <Box direction="row" gap="small">
```

Comment:
Should we remove this for the time being since we still need to add icon.onStrong? The color doesn't come through since it doesn't exist yet.